doc/generated/tools.gen

```diff
@@ -139,9 +139,89 @@ Set construction variables for cygwin linker/loader.
     <term>default</term>
     <listitem>
 <para xmlns="http://www.scons.org/dbxsd/v1.0">
-Sets variables by calling a default list of Tool modules
-for the platform on which SCons is running.
+Sets <literal xmlns="http://www.scons.org/dbxsd/v1.0">construction variables</literal> for a default list of Tool modules.
+Include <emphasis role="bold">default</emphasis>
```

Comment:
maybe changes this to "append" rather than include?

Reply:
Seems like a lot of examples look like `tools=['default', 'foo']` ... which isn't "append" since it's at the beginning. Language nit-picking, I know.